ovs/lib/graphite.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2018 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+import time
+from ovs.extensions.generic.graphiteclient import GraphiteClient
+from ovs.extensions.generic.configuration import Configuration
+
+
+class GraphiteController():
+    """
+    Graphite Controller that sends or does not send, depending on the saved config setting
+    """
+
+    def __init__(self, database=None):
+        celery_scheduling = Configuration.get(key='/ovs/framework/scheduling/celery', default={})
+        self._send_statistics = any(celery_scheduling.get(key) is not None for key in ['ovs.stats_monkey.run_all', 'alba.stats_monkey.run_all'])
+        if self._send_statistics:
```

Comment:
Set self._client = None here too. If self._send_statistics would be False, the class would have one property less otherwise.